misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -68,6 +69,9 @@ public CoverageReport(Path sourceJarPath, Path targetDirPath, String orgName, St
         this.classesDirectory = projectDir.resolve(TesterinaConstants.BIN_DIR);
         this.executionDataFile = projectDir.resolve(TesterinaConstants.EXEC_FILE_NAME);
         this.execFileLoader = new ExecFileLoader();
+
+        this.jarCache = targetDirPath.resolve("caches/jar_cache/").resolve(orgName);
```

Comment:
Lets resolve each child directory. i.e. `targetDirPath.resolve("caches").resolve("jar_cache").resolve(orgName)`

Reply:
Applies to L74 as well.

Reply:
Changed to resolve each child directory